src/dosinst.c

```diff
@@ -1899,7 +1903,7 @@ build_shortcut(
     char	executable_path[BUFSIZE];
     char	link_name[BUFSIZE];
 
-    sprintf(executable_path, "%s\\%s", installdir, exename);
+    snprintf(executable_path, sizeof(executable_path), "%s\\%s", installdir, exename);
```

Comment:
`snprintf` might not be able to use in old MSVC.
`_snprintf` might be used instead but the specification is slightly different.